### DISCUSSION TITLE
Zombie app suggestions

### DISCUSSION THREAD

**Discussion by dtetreau251** [Vets-Who-Code/march2021#68] [Ideas]
- [x] 1. Media Queries, reorganize CSS, and footer styling (Steve claimed these), 

- [x] 2. Modal and sound for jump scare (David claimed this. Great idea Joe!) 

3. About.html page 

- [x] 4. window alert() method (maybe one that automatically pops up warning all those who enter) 

- [x] 5. something with innerHTML 

6. something with window prompt() (Mario Assigned)

7. A form (maybe collect visitor info?) 

8. links to other sites 

9. Add a table

**Comment by HerndonC** [comment]
10. Make a tool bar, similar to something I've worked on before, with different tabs called Zombie Tools. Use JS like the zombie radar to make fake "tools"
11. An onload alert that asks if you are afraid of zombies, if they say no, continue to page, if they say yes, change the page to something like puppies or something truly terrifying, like MLP
12. For the zombie radar button, if you hit cancel, add some pictures of grave and put like... an impending doom feel on it, if they say yes, make it a big bright message saying to evacuate. (If anyone wants to get real fancy with it, we can do a modal window or embed a map that automatically searches for nearby evacuation centers)
13. I was probably going to make an about page for us that links to our capstones/prework stuff.

**Reply by dtetreau251** [reply]
The modal is complete along with window prompt() and innerHTML

**Reply by TCoder920** [reply]
🤔 14. Change the zombie radar button into a radar gif that prompts the same message?

**Reply by dtetreau251** [reply]
i changed the radar button into a clickable card that has a radar icon on it.

**Comment by dtetreau251** [comment]
Sounds like fun !
